include/class-polylang.php

```diff
@@ -165,19 +168,22 @@ public static function define_constants() {
 	 */
 	public function init() {
 		self::define_constants();
-		$options = get_option( 'polylang' );
+
+		// Plugin options.
+		add_action( 'pll_init_options_for_blog', array( Options_Registry::class, 'register_options' ) );
```

Comment:
Couldn't this be moved to `Options`?

Reply:
It could but I'm not sure it should: registering options for PLL would be in `Options` then, but registering options for PLL Pro would be outside.
I have no strong feelings about this though, so we can do whatever you prefer.